addons/binding/org.openhab.binding.zway/src/main/java/org/openhab/binding/zway/handler/ZWayBridgeHandler.java

```diff
@@ -490,6 +495,32 @@ protected ZWayBridgeConfiguration getZWayBridgeConfiguration() {
         return mConfig;
     }
 
+    /*******************************
```

Comment:
I personally dislike banners :-)

Reply:
But I see that we used it before

Reply:
Okay. Left it. All other points have been incorporated